README.md

```diff
@@ -44,26 +44,26 @@ Based on those 4 families, the MiddlewareListServiceProvider provides a SPL Prio
 
 The service provider defines 16 constants you can use to insert a middleware at a given point:
 
-- `MiddlewareOrder::UTILITY_EARLY` (-50)
+- `MiddlewareOrder::UTILITY_EARLY` (50)
 - `MiddlewareOrder::UTILITY` (0)
-- `MiddlewareOrder::UTILITY_LATE` (50)
-- `MiddlewareOrder::ROUTER_EARLY` (950)
+- `MiddlewareOrder::UTILITY_LATE` (-50)
+- `MiddlewareOrder::ROUTER_EARLY` (1050)
 - `MiddlewareOrder::ROUTER` (1000)
-- `MiddlewareOrder::ROUTER_LATE` (1050)
-- `MiddlewareOrder::PAGE_NOT_FOUND_EARLY` (1950)
+- `MiddlewareOrder::ROUTER_LATE` (950)
+- `MiddlewareOrder::PAGE_NOT_FOUND_EARLY` (2050)
 - `MiddlewareOrder::PAGE_NOT_FOUND` (2000)
-- `MiddlewareOrder::PAGE_NOT_FOUND_LATE` (2050)
+- `MiddlewareOrder::PAGE_NOT_FOUND_LATE` (1950)
```

Comment:
Something is wrong here....

Page not found should act last (after the router that itself should act after the utility middleware).

So the "pagenotfound" routers should definitely have the smallest priority, and so on...